avocado/plugins/remote.py

```diff
@@ -106,6 +106,7 @@ def _check_required_args(args, enable_arg, required_args):
     def run(self, args):
         if self._check_required_args(args, 'remote_hostname',
                                      ('remote_hostname',)):
+            args.loaders = ['remote']
```

Comment:
Well this is actually not necessary as you can simply:

```python
from avocado.core.loader import loader

loader.register_plugin(loaders.RemoteLoader)
```

which would be IMO clearer.